fs/inode/inode.h

```diff
@@ -229,9 +229,12 @@ int inode_find(FAR struct inode_search_s *desc);
  *
  * Input Parameters:
  *   inode   - The inode of interest
- *   buf     - The caller provide location in which to return information
+ *   buf     - The caller provided location in which to return information
```

Comment:
```suggestion
 *   buf     - The caller-provided location in which to return information
```
Sorry for being pedantic, but missed the hyphen in `caller-provided`.

Reply:
Done.